packages/daemon/test/test-endo.js

```diff
@@ -29,8 +30,8 @@ const locator = {
   cachePath: path.join(dirname, 'tmp/cache'),
   sockPath:
     process.platform === 'win32'
-      ? raw`\\?\pipe\endo-test.sock`
-      : path.join(dirname, 'tmp/endo.sock'),
+      ? raw`\\?\pipe\endo-test-${process.pid}.sock`
+      : path.join(os.tmpdir(), `endo-test-${process.pid}.sock`),
```

Comment:
Using an OS tmpdir makes it harder to inspect the logs when the tests fail. If we go this route, there at least needs to be a `t.log` so the developer can rendezvous with the dump, but I’m inclined to keep it local. There’s a corresponding entry in `.gitignore`.

Reply:
Isn't this just the socket path, and nothing else?  Or is there some bizarre derivation of the log path from the socket path I'm missing.

Reply:
Oh, I see. You are correct. This only affects the location of the socket or named pipe, not the log dump.